test/unit/printer/sourcemaps.spec.ts

```diff
@@ -296,13 +296,14 @@ test("Inline sourcemaps", () => {
     const file = util
         .testFunction(code)
         .setOptions({ inlineSourceMap: true })
+        .disableSemanticCheck() // TS5053: Option 'sourceMap' cannot be specified with option 'inlineSourceMap'.
```

Comment:
`file.luaSourceMap` (which is used for comparison with inlined source map content) is parsed from generated `.map` file. Previously it did work because `TranspiledFile.sourceMap` property existed even without `sourceMap: true` option

Reply:
Could you add this reasoning to the comment here? (ie 'We need the source map to compare to for the test')